indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -482,12 +502,13 @@ sealed class Asn1Structure(
             postfix = "\n" + (" " * indent) + "}"
         ) { it.prettyPrint(indent + 2) }
 
-    override fun contentToString(): String = (if (isSorted) " SORTED " else "") + ", children=$children"
+    override fun contentToString(): String =
+        (if (shouldBeSorted) " should be SORTED " else "") + (if (isActuallySorted) " (is acutely SORTED) " else " (is NOT SORTED) ") + ", children=$children"
 
     override fun hashCode() = 31 * super.hashCode() + children.hashCode()
 
     /**
-     * the [isSorted] flag has no bearing on equals!
+     * the [doSort] flag has no bearing on equals!
```

Comment:
shouldBeSorted?

Reply:
is does sort it if you set it to true…

Reply:
nvm. it is more consitent that way

Reply:
clarification: these are different flags; `shouldBeSorted` is the `val`, `doSort` is the constructor parameter that indicates whether we are parsing